src/app/pages/HomePage/ParaTimeSelector/types.ts

```diff
@@ -0,0 +1,14 @@
+export enum ParaTimeSelectorStep {
+  /**
+   * Animation in progress
+   */
+  ANIMATING,
```

Comment:
at some point we will have to unify enums, capitalize vs camelcase, using os string enums etc but we don;t have code standards setup yet